libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -362,4 +403,18 @@ export class DrivingLicenseService {
       teacherName,
     }
   }
+
+  async getDrivingSchool(nationalId: string): Promise<DrivingSchool | null> {
+    const result: HefurLokidOkugerdiDto = await this.drivingLicenseApi.apiOkuskirteiniKennitalaFinishedokugerdiGet(
```

Comment:
This function name would be better to read as
`apiOkuSkirteiniKennitalaFinishedOkugerdiGet`